snap/local/pebble-wrapper

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -eu
+
+# The default Pebble directory should sit within the $SNAP_USER_DATA because:
+# - access to the system's /var/lib/pebble/default isn't guaranteed, and
+# - the default $PEBBLE path should also be removed when removing the snap.
+DEFAULT_PEBBLE_DIRECTORY="${SNAP_USER_DATA}/var/lib/pebble/default"
```

Comment:
Does it need to be under `/var/lib/pebble/default` *within* the Snap user data directory? What about just `DEFAULT_PEBBLE_DIRECTORY="${SNAP_USER_DATA}"`?

This would also avoid the need for the user to create the directory or use `--create-dirs`, because (I presume) Snap ensures it exists.

Reply:
This is possible and an easy change. The only thing I'm not too comfortable with is the user-facing information. I.e. `pebble help` will print

```
Set the PEBBLE environment variable to override the configuration directory 
(which defaults to /var/lib/pebble/default). Set PEBBLE_SOCKET to override 
the unix socket used for the API (defaults to $PEBBLE/.pebble.socket).
```

So that's why keeping the `/var/lib/pebble/default` as a path suffix sounded more natural to me, despite it requiring the explicit creation of said path (which is actually what you'd expect if you were installing Pebble from source or via the existing release binaries).

So having said that, if you still think it would be better to just have `$SNAP_USER_DATA`, I'll change it.

Reply:
I do think it'd be better, but I see your concern. How about we tweak the help message while we're at it. Something like this (with better wrapping):

```
Set the PEBBLE environment variable to override the configuration directory (which
defaults to /var/lib/pebble/default, or to SNAP_USER_DATA if being used via the Snap).
Set PEBBLE_SOCKET to override the unix socket used for the API (defaults to $PEBBLE/.pebble.socket).
```

Or perhaps say `or to $HOME/snap/pebble/<rev> if being used via the Snap` -- I'm not sure which is better from a Snap user point of view.